src/sentry/api/serializers/models/filechange.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+
+import six
+
+from sentry.api.serializers import Serializer, register
+from sentry.models import CommitFileChange
+from sentry.api.serializers.models.release import get_users_for_commits
+
+
+@register(CommitFileChange)
+class CommitFileChangeSerializer(Serializer):
+    def get_attrs(self, item_list, user):
+        commits = [f.commit for f in item_list]
+        author_objs = get_users_for_commits(commits)
+        result = {}
+        for item in item_list:
+            result[item] = {
+                'user': author_objs.get(item.commit.author_id, {})
+            }
+
+        return result
+
+    def serialize(self, obj, attrs, user):
+        return {
+            'id': six.text_type(obj.id),
+            'org_id': obj.organization_id,
+            'author': attrs.get('user', {}),
+            'commit_message': obj.commit.message,
```

Comment:
We need to make sure this doesn't create an O(n) query by resolving `obj.commit`. I'd say we manage these relationships through `attrs` and fetch them from there instead of hoping it's resolved on the `obj` itself.